schema/server/list.json

```diff
@@ -17,10 +17,16 @@
                 "description": "Task parameters, used when running and for querying",
                 "type": ["null", "object"]
             },
-
-            "custom": {
```

Comment:
Was custom not used for anything? Why remove it?

Reply:
Do we use PackitPacket anywhere else do you know? Or is that just for the API response?

Reply:
Yup as discussed yesterday custom was not being used at all by Packit. And its only being removed from that api response object (PackitPacket)

Reply:
We might this back in at some point but OK to pull it out for now; this is where the orderly specific metadata would go